be/src/env/env_s3.cpp

```diff
@@ -77,15 +77,20 @@ StatusOr<std::unique_ptr<RandomAccessFile>> EnvS3::new_random_access_file(const
     std::string object = path.substr(namenode.size(), path.size() - namenode.size());
     Aws::Client::ClientConfiguration config;
     config.scheme = Aws::Http::Scheme::HTTP;
-    if (!config::aws_s3_endpoint.empty()) {
-        config.endpointOverride = config::aws_s3_endpoint;
+    if (is_oss_path(namenode.c_str())) {
```

Comment:
Is this our own OSS URL format? It does not appear to be a standard OSS URL format, not is it a standard S3 URL format.

A URI star with with `oss://` is neither a valid S3 URL nor a valid OSS URL.  The virtual-hosted–style accessing of S3 is compatible 
FYI:
https://help.aliyun.com/document_detail/31834.html
https://docs.aws.amazon.com/AmazonS3/latest/userguide/access-bucket-intro.html

Reply:
It's a standard OSS path. URL format with sheme "http" you mentioned is for external access usually.

For SDK access, we should use standard access path like "s3a://" for aws or "oss://" for aliyun